packages/ember-flight-icons/src/components/flight-icon.ts

```diff
@@ -6,17 +6,49 @@
 import Component from '@glimmer/component';
 import { guidFor } from '@ember/object/internals';
 import { assert } from '@ember/debug';
-
 import { iconNames } from '@hashicorp/flight-icons/svg';
 
-export default class FlightIconComponent extends Component {
-  constructor() {
-    super(...arguments);
+import type { IconName } from '@hashicorp/flight-icons/svg';
+
+export interface FlightIconSignature {
+  Args: {
+    /**
+     * The color of the icon. Defaults to `currentColor`.
+     */
+    color?: string;
+    /**
+     * The name of the icon to display.
+     */
+    name?: IconName;
```

Comment:
Should this be required? 🤔

Reply:
I have tried that but, several consuming components are not happy about this change, I could potentially look into fixing the consuming side but since `Hds::Icon` is coming pretty soon I would say we have an assert segment that would catch potential errors, this should be enough till the new `Hds::Icon` is shipped

Reply:
OK, makes sense. 👍